src/app/app.module.ts

```diff
@@ -33,9 +37,15 @@ import { PersonProfileComponent } from './person-profile/person-profile.componen
     MatGridListModule,
     MatToolbarModule,
     MatIconModule,
-    HttpClientModule
+    MatDialogModule,
+    HttpClientModule,
+    FormsModule,
+    MatDatepickerModule,
+    MatNativeDateModule
   ],
   providers: [PositiveCaseService, UserService],
-  bootstrap: [AppComponent]
+  bootstrap: [AppComponent],
+  entryComponents: [AddCaseDialogueComponent],
+  schemas: [CUSTOM_ELEMENTS_SCHEMA]
```

Comment:
I don't understand where this CUSTOM_ELEMENTS_SCHEMA is used. Did I miss it somewhere?

Reply:
It is declared in the schema section of app.module.ts. It is necessary for some Material html tags in the add-positive-case component.